reference/7.0/Microsoft.PowerShell.Utility/Get-Error.md

```diff
@@ -161,4 +161,4 @@ Output in a **PSExtendedError** object.
 
 ## RELATED LINKS
 
-[about_Try_Catch_Finally](about_Try_Catch_Finally.md)
+[about_Try_Catch_Finally](../microsoft.powershell.Microsoft.PowerShell.Core/About/about_Try_Catch_Finally.md)
```

Comment:
Should be 
`[about_Try_Catch_Finally](../Microsoft.PowerShell.Core/About/about_Try_Catch_Finally.md)`